scheds/rust/scx_layered/src/main.rs

```diff
@@ -88,6 +94,13 @@ const NR_LLC_LSTATS: usize = bpf_intf::llc_layer_stat_id_NR_LLC_LSTATS as usize;
 
 const NR_LAYER_MATCH_KINDS: usize = bpf_intf::layer_match_kind_NR_LAYER_MATCH_KINDS as usize;
 
+static NVML: OnceCell<Nvml> = OnceCell::new();
+
+#[inline]
```

Comment:
Let's let the compiler decide this.

Reply:
Will try and make this change -- this is boilerplate to make nvml only be initialized once because that can be heavy.

Reply:
Oh, I was just talking about dropping `inline`.